core/inspector.rs

```diff
@@ -438,6 +438,7 @@ struct InspectorWakerInner {
   isolate_handle: v8::IsolateHandle,
 }
 
+// SAFETY: TODO
```

Comment:
These are now required by https://github.com/denoland/deno/blob/ee2f4e745c33797ac9fd20571e77cef73ea585bf/.cargo/config.toml#L25

Not sure what a good safety warning would be for all the cases.